js/main.js

```diff
@@ -1,2 +1,34 @@
 /* global data */
 /* exported data */
+
+var $imageInput = document.querySelector('#image-url');
+var $image = document.querySelector('#image');
+
+function addImage(event) {
+  $image.setAttribute('src', $imageInput.value);
+}
+
+$imageInput.addEventListener('blur', addImage);
+
+var $newEntryForm = document.querySelector('#entry-form');
+
+function clickSave(event) {
+  event.preventDefault();
+  var titleValue = $newEntryForm.elements.title.value;
+  var notesValue = $newEntryForm.elements.notes.value;
+  var imageValue = $newEntryForm.elements.url.value;
+  var entryObj = {
+    URL: imageValue,
+    title: titleValue,
+    notes: notesValue,
+    entryId: data.nextEntryId
+  };
+  data.nextEntryId++;
+  data.entries.push(entryObj);
```

Comment:
You want to add to the beginning of the array, not the end. You will need a different method than `push`

Reply:
Changed this to unshift